packages/dropdowns/src/styled/field/StyledSelect.ts

```diff
@@ -83,7 +83,8 @@ export interface IStyledSelectProps {
 export const StyledSelect = styled(FauxInput).attrs<IStyledSelectProps>(props => ({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
-  'aria-invalid': isInvalid(props.validation)
+  'aria-invalid': isInvalid(props.validation),
+  theme: props.theme
```

Comment:
I'm a bit stumped by this one. Is it because the object being extended is a high-abstraction vs. styled component? Either way, this seems to fix the issue of theme not being applied to the underlying `StyledFauxInput`.

Reply:
I tried removing `props.theme` (removal of line 87). I console logged `theme` throughout this file and the `theme` was available. Locally, after I removed that line 87, the UI seems to be the same as the "after" screenshots posted in the PR. Would you be able to confirm if adding `props.theme` here is needed?

Reply:
Did you alter the theme and wrap the `Select` with a custom theme provider? Without this addition, you would get a result similar to the **Before** shown in the PR description.

Reply:
I only removed `props.theme` on line 87, and I got a similar result to the *After* screenshot.